### DISCUSSION TITLE
How to share/open a connection?

### DISCUSSION THREAD

**Discussion by gruevy** [vladmandic/sdnext#323] [General]
In the main automatic1111, you put the flag to share in the webui-user.bat file. There isn't one. Where do I put it? How does that work now? 

To create a public link, set `share=True` in `launch()`.

**Comment by vladmandic** [comment]
just create your `webui-user.bat` with a single line
```
@webui --share
```
there is no point of shipping `webui-user.bat` with the app when it would be empty.
